solidity/contracts/bridge/Bridge.sol

```diff
@@ -53,106 +54,24 @@ import "../bank/Bank.sol";
 ///         balances in the Bank.
 /// @dev Bridge is an upgradeable component of the Bank.
 ///
-/// TODO: All wallets-related operations that are currently done directly
-///       by the Bridge can be probably delegated to the Wallets library.
-///       Examples of such operations are main UTXO or pending redemptions
-///       value updates.
+// TODO: All wallets-related operations that are currently done directly
+//       by the Bridge can be probably delegated to the Wallets library.
+//       Examples of such operations are main UTXO or pending redemptions
+//       value updates.
 contract Bridge is Ownable, EcdsaWalletOwner {
     using BridgeState for BridgeState.Storage;
     using Deposit for BridgeState.Storage;
     using Sweep for BridgeState.Storage;
+    using Redeem for BridgeState.Storage;
```

Comment:
Seems all libraries are nouns. Maybe we can call it `Redemption` instead of `Redeem`?

Reply:
Yeah, good point. I could feel something was wrong with this name but I had no idea what. Know I know, thanks! 😆

Reply:
Though... can I rename it in the final cleanup PR? Would prefer to avoid conflicts. I will add a TODO.

Reply:
Sure!

Reply:
Renamed in https://github.com/keep-network/tbtc-v2/pull/211.